_posts/2018-04-19-working-group-invitation.md

```diff
@@ -0,0 +1,55 @@
+
+You may recall as part of [the 2018 roadmap (RFC 1728)](https://github.com/rust-lang/rfcs/pull/2314) and other community efforts multiple working groups have been formed around different topics. Yann, 1aim's co-founder, reached out to these working groups and spoke with them about how to best support the teams and support contributors.
+
+## What is a Working Group?
+
+[Working groups](https://internals.rust-lang.org/t/announcing-the-2018-domain-working-groups/6737) are assemblies of contributors. Working Groups work directly with the Core Team and focus on the imporving the end-to-end user experience of using Rust in their respecitve domain. That work can involve the language, compiler, libraries, tools, documentation, discoverability, and more! The overall effort is organized by Aaron Turon (`aturon`).
+
+* Networking services, lead by `withoutboats` and `cramertj`
+* [WebAssembly](https://internals.rust-lang.org/t/come-join-the-rust-and-webassembly-working-group/6845), lead by `fitzgen`
+* [CLI apps](https://internals.rust-lang.org/t/announcing-the-cli-working-group/6872), lead by `killercup`
+* [Embedded devices](https://internals.rust-lang.org/t/announcing-the-embedded-devices-working-group/6839), lead by `japaric`
+
+We have also seen other groups forming such as the [Formal Verification](https://internals.rust-lang.org/t/announcing-the-formal-verification-working-group/7240), [Traits](https://internals.rust-lang.org/t/announcing-traits-working-group/6804) and [Rust Compiler Performance](https://internals.rust-lang.org/t/rust-compiler-performance-working-group/6934) working groups. Maybe we'll even see more form!
+
+You can also contact your **lead** or **Yann from 1aim (`Yann` on irc.mozilla)** directly with questions.
+
+# Sponsorships
+
+In their ongoing support of the RustFest community 1aim has offered to support **17 community members** in coming to RustFest! 1aim will be providing the following sponsorships:
+
+* 12 **Working Group** Sponsorships
+    * To folks who are active contributors to the Working Groups.
+    * Nominated by working group leads. (*No, they can't nominate themselves!* :))
+* 5 **Diversity** Sponsorships
+    * In addition to those that RustFest & Diversity Sponsor Donations support
+    * Selected from previous **[Diversitytickets.org event](https://diversitytickets.org/events/152)**.
+
+**What do you get?**
+* One RustFest Paris **ticket**.
+* **Hotel accommodation** from Sept. 29 to Oct. 2. (Provided by 1aim)
+* Up to **150€** for EU and **250€** for non-EU participants toward your travel expenses (reimbursed post-conference, pending receipts)
+
+**How do you participate?**
+If are nominated by a WG lead, **sign up [here](https://goo.gl/forms/A1hrLjA81kiBidff2)** until the **29th of April 2018**. If there are more than 12 nominated applications, the winners will be randomly drawn from the pool. You will have one week to accept your invitation or it will go to someone else.
```

Comment:
> If **you** are nominated

Reply:
Is 29th enough time? It's already quite late and I'd like to see this done quickly, but do we need to give people more time?

Reply:
I don't think it's enough time, but I think most of the people who would qualify have already been participating.